src/main/java/com/hotels/beeju/BeejuJUnitRule.java

```diff
@@ -132,29 +90,21 @@ protected void after() {
    * @return the name of the JDBC driver class used to access the database.
    */
   public String driverClassName() {
-    return driverClassName;
-  }
-
-  /**
-   * @return a copy of the {@link HiveConf} used to create the Hive Metastore database. This {@link HiveConf} should be
-   *         used by tests wishing to connect to the database.
-   */
-  public HiveConf conf() {
```

Comment:
maybe keep this method and delegate to `core.conf()` just like the other methods.

Reply:
ah yeah sure, should be consistent, i'll put that back